hummingbird/ml/operator_converters/onnx_tree_ensembles.py

```diff
@@ -0,0 +1,186 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+Converters for ONNX-ML tree-ensemble models.
+"""
+
+import numpy as np
+from onnxconverter_common.registration import register_converter
+
+from . import constants
+from ._gbdt_commons import convert_gbdt_classifier_common, convert_gbdt_common
+from ._tree_commons import TreeParameters, get_parameters_for_tree_trav_common, convert_decision_ensemble_tree_common
```

Comment:
I don't see where either of these is used in this file, unless I am missing it.  But it passed lint/flake8...so maybe I don't understand something.

Reply:
Yea I don't know what is going on with flake8. I removed the unnecessary one.